bin/maf/generate_ss

```diff
@@ -119,6 +121,7 @@ def generate_ss_commands(dbfiles=None, pops=None, start_mjd=60218.0, split=False
                     s3 = (
                         f"run_moving_fractions --workDir {run}_ss"
                         f" --metadata {objtype}"
+                        f" --hMark {default_plots_values['Hmark']} "
```

Comment:
Hmm, I thought the hMark should have been recorded in the metric file when it was configured at step1 (from the movingObjectBatch), so wouldn't have had to be repeated here. But it's also not hurtful. 
Worth noting - the values returned from ss_population_defaults are not just for the plots. 
Your renaming of the function is much improved, but I'm going to update the returned values to indicate the 'not just plots' part as well.

Reply:
(or maybe I won't bother .. it looks like it is only being used for the plots here)